### ISSUE TITLE
I want to know how your code differs from the original version of the pytorch code. 

### ISSUE THREAD

**Issue by 394781865** [MayankSingal/VideoExpansion#7]
As far as I know you replaced resnet with vgg. in addition, 
your Overall Loss：
Loss = 0.8*loss_reconstruction + 0.005*loss_perceptual + 0.4*loss_warping + loss_smooth
the original version of the pytorch code：
loss = 204 * recnLoss + 102 * warpLoss + 0.005 * prcpLoss + loss_smooth
